solana-common/src/pb/sf.solana.type.v1.rs

```diff
@@ -0,0 +1,256 @@
+// @generated
```

Comment:
I think that now that we use the one from `substreams-solana`, we don't need that.